ui/imports/shared/views/chat/ProfileContextMenu.qml

```diff
@@ -1,98 +1,43 @@
-    }
-    readonly property int incomingVerificationStatus: {
-        if (root.selectedUserPublicKey === "" || root.isMe || !root.isContact) {
-            return 0
-        }
-        return contactDetails.incomingVerificationStatus
-    }
-    readonly property bool hasPendingContactRequest: {
-        return !root.isMe && root.selectedUserPublicKey !== "" &&
-                contactDetails.contactRequestState === Constants.ContactRequestState.Received
-    }
-    readonly property bool hasActiveReceivedVerificationRequestFrom: {
-        if (!root.selectedUserPublicKey || root.isMe || !root.isContact) {
-            return false
-        }
-        return contactDetails.incomingVerificationStatus === Constants.verificationStatus.verifying ||
-                contactDetails.incomingVerificationStatus === Constants.verificationStatus.verified
-    }
-    readonly property bool isVerificationRequestSent: {
-        if (!root.selectedUserPublicKey || root.isMe || !root.isContact) {
-            return false
-        }
-        return root.outgoingVerificationStatus !== Constants.verificationStatus.unverified &&
-                root.outgoingVerificationStatus !== Constants.verificationStatus.verified &&
-                root.outgoingVerificationStatus !== Constants.verificationStatus.trusted
-    }
-    readonly property bool isTrusted: {
-        if (!root.selectedUserPublicKey || root.isMe || !root.isContact) {
-            return false
-        }
-        return root.outgoingVerificationStatus === Constants.verificationStatus.trusted ||
-                root.incomingVerificationStatus === Constants.verificationStatus.trusted
-    }
-
-    readonly property bool userTrustIsUnknown: contactDetails && contactDetails.trustStatus === Constants.trustStatus.unknown
-    readonly property bool userIsUntrustworthy: contactDetails && contactDetails.trustStatus === Constants.trustStatus.untrustworthy
-    readonly property bool userIsLocallyTrusted: contactDetails && contactDetails.trustStatus === Constants.trustStatus.trusted
+    property int trustStatus: Constants.trustStatus.unknown
+    property int contactType: Constants.contactType.nonContact
+    property bool ensVerified: false
+    property int onlineStatus: Constants.onlineStatus.unknown
+    property bool hasLocalNickname: false
+    property int profileType: Constants.profileType.regular
 
     signal openProfileClicked(string publicKey)
     signal createOneToOneChat(string communityId, string chatId, string ensName)
+    signal reviewContactRequest(string publicKey)
+    signal sendContactRequest(string publicKey)
+    signal editNickname(string publicKey)
+    signal removeNickname(string publicKey, string displayName)
+    signal unblockContact(string publicKey)
+    signal markAsUntrusted(string publicKey)
+    signal removeTrustStatus(string publicKey)
+    signal removeContact(string publicKey)
+    signal blockContact(string publicKey)
 
     onClosed: {
```

Comment:
It's a good chance to remove this `onClosed` handler. It's not needed and potentially a source of bugs because it's breaking the bindings. We're currently destroying the menu on close anyways.